common/authorization/authorizer.go

```diff
@@ -38,8 +38,7 @@ const (
 type (
 	// Attributes is input for authority to make decision.
 	// It can be extended in future if required auth on resources like WorkflowType and TaskQueue
-	Attributes struct {
-		Actor     string
+	CallTarget struct {
```

Comment:
Just wanted to double check: looks like we are changing the names of public structs and interfaces here. Is it okay to do this for this specific scenario?

Reply:
Yes, I believe this is the case where we should do it. `Attributes` is a confusing name, and it includes `Actor` that is not enough to convey properties of the caller IMO.